net/scripts/gce-provider.sh

```diff
@@ -161,7 +159,6 @@ cloud_CreateInstances() {
   declare networkName="$1"
   declare namePrefix="$2"
   declare numNodes="$3"
-  declare enableGpu="$4"
   declare machineType="$5"
```

Comment:
update the argument indexes

Reply:
let me know if you still want me to update indices with this in mind